lxd/container.go

```diff
@@ -738,6 +720,7 @@ func containerCreateInternal(d *Daemon, args containerArgs) (container, error) {
 	args.CreationDate = dbArgs.CreationDate
 	args.LastUsedDate = dbArgs.LastUsedDate
 
+	// Setup the container struct and finish creation (storage and idmap)
```

Comment:
I meant the storage volume (on disk) is created later. This read like you're saying that this function creates the storage volume on disk. It really is not that important.